src/lib/scheduler/topology.cpp

```diff
@@ -21,7 +21,15 @@ Topology& Topology::current() {
   return instance;
 }
 
-Topology::Topology() { init_default_topology(); }
+Topology::Topology()
+#if HYRISE_NUMA_SUPPORT
+  : _numa_num_configured_nodes(numa_num_configured_nodes())
```

Comment:
Configured sounds more like user-defined, but if I understand you correctly, this is the number of hardware nodes. Maybe `_number_of_hardware_nodes`? Also, shouldn't this be static?